snap/info.go

```diff
@@ -753,7 +753,7 @@ type AppInfo struct {
 
 // ScreenshotInfo provides information about a screenshot.
 type ScreenshotInfo struct {
-	URL    string `json:"url"`
+	URL    string `json:"url,omitempty"`
```

Comment:
Note this is technically an API break. This isn't an issue in snapd-glib (we treat missing fields as empty) but I don't know if any other clients will care.